src/webapp/hooks/useRoutes.ts

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { generatePath, useHistory } from "react-router-dom";
+import { FormType } from "../pages/form-page/FormPage";
+
+export enum RouteNames {
+    CREATE_FORM = "CREATE_FORM",
+    EDIT_FORM = "EDIT_FORM",
+    EVENT_TRACKER = "EVENT_TRACKER",
+    IM_TEAM_BUILDER = "IM_TEAM_BUILDER",
+    INCIDENT_ACTION_PLAN = "INCIDENT_ACTION_PLAN",
+    RESOURCES = "RESOURCES",
+    DASHBOARD = "DASHBOARD",
+}
+
+export const routes: Record<RouteNames, string> = {
+    [RouteNames.CREATE_FORM]: "/create/:formType",
+    [RouteNames.EDIT_FORM]: "/edit/:formType/:id",
+    [RouteNames.EVENT_TRACKER]: "/event-tracker",
+    [RouteNames.IM_TEAM_BUILDER]: "/incident-management-team-builder",
+    [RouteNames.INCIDENT_ACTION_PLAN]: "/incident-action-plan",
+    [RouteNames.RESOURCES]: "/resources",
+    [RouteNames.DASHBOARD]: "/",
+};
```

Comment:
react-router-dom has some fancy string template processing, to take advantage of it, add `as } const;`